test/testcafe/spec-en-leaks/EnglishLeaks_spec.js

```diff
@@ -38,6 +38,18 @@ const ignoredMocks = [
   'error-429-api-limit-exceeded.json', // Gen3 has english leak on UI. But in real world this response can be received on polling only, no english leak in this case.
 ];
 
+const ignoredLeaks = {
+  'identify-with-webauthn-launch-authenticator.json': [
+    'This is an invalid domain.', // browser webauthn error
```

Comment:
TestCafe3 does not produce "Security key or biometric authenticator is not supported on this browser."
But for Gen2 browser returns error `This is an invalid domain.` which is en leak